tests/DwitTech.NotificationService.WebApi.Tests/DwitTech.NotificationService.WebApi.Tests.csproj

```diff
@@ -9,6 +9,11 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="MailKit" Version="3.5.0" />
```

Comment:
what is this MailKit everywhere here for? If not in use, please remove it.

Reply:
> what is this MailKit everywhere here for? If not in use, please remove it.

Resolved